jetsocat/Cargo.toml

```diff
@@ -36,7 +36,7 @@ seahorse = "2.2"
 humantime = "2.1"
 
 # async
-tokio = { version = "1.41", features = ["io-std", "io-util", "net", "fs", "signal", "time", "rt", "sync", "process", "rt-multi-thread", "macros"] }
+tokio = { version = "1.38", features = ["io-std", "io-util", "net", "fs", "signal", "time", "rt", "sync", "process", "rt-multi-thread", "macros"] }
```

Comment:
You probably got a problem when you merged changes from master, I see some things being reverted here

Reply:
updated